examples/helloworld/publisher.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright(c) 2006 to 2020 ADLINK Technology Limited and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+ * v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+#include <stdio.h>
+#include <stdlib.h>
+
+#ifdef _WINDOWS
```

Comment:
These days there is std::thread::sleep_for and with std::chrono providing nice literals so that one can write `std::thread::sleep_for(20ms)` (see https://en.cppreference.com/w/cpp/thread/sleep_for). I think that'd be a lot nicer than this old hack.

Reply:
The literals are only available since `C++14`. I opted for `std::chrono::milliseconds(xx)` instead, which is available since `C++11`.